cmd/tiller/tiller.go

```diff
@@ -141,6 +147,18 @@ func start() {
 
 		env.Releases = storage.Init(secrets)
 		env.Releases.Log = newLogger("storage").Printf
+	case storageSQL:
+		sqlDriver, err := driver.NewSQL(
+			*sqlDialect,
```

Comment:
It would be nice if we validated that the dialect chosen was valid or not. For example, I could enter `postgresql` or `postgres`. Without validation we cannot inform the user which dialects tiller supports or not. As far as I can tell we only accept postgresql as a valid dialect.

Reply:
Totally right, we added some extra validation on the storage driver in the `NewSQL()` method. 

As you mentioned above, since we're importing `pq` only, the current implementation is limited to postgres. That totally makes sense.